sourcegraph/sourcegraph
# tabs persist on refresh

Before, tabs did not persist on refresh in the code monitors page. Now, they do.

## Test plan
Manually tested in UI - clicking and refreshing
Manually tested that the URL params change when active tab changes
Manually tested visiting the URL with params already filled into make sure that the right tab is set when visiting
